### PR TITLE
Add Deepsight-related Organizer columns

### PR DESCRIPTION
**Pull request by bjacobel**
Refs https://github.com/DestinyItemManager/DIM/issues/10184

Adds:
* An Organizer column for "Deepsight level" (ie: crafted weapon level)
  image
* An Organizer column for "Deepsight harmonizable" (ie: can a Harmonizer be used to extract Deepsight crafting progress)
  image

Let me know if there's manual work that needs doing for the new i18n entries? I wasn't sure how that stuff works.

### CODE REVIEW

**Comment by bjacobel** [comment]
Just realizing now that my branch was _way_ out of date, will need a little time to rebase this...

File: config/i18n.json [hunk 1]
```diff
@@ -1029,6 +1029,8 @@
       "CustomTotal": "Custom Total",
       "MasterworkTier": "Masterwork Tier",
       "MasterworkStat": "Masterwork Stat",
+      "DeepsightLevel": "Deepsight Level",
```
Comment (bhollis) [inline config/i18n.json:1032]: This could be just "Level", since there aren't other types of Level for weapons (so far)

File: config/i18n.json [hunk 2]
```diff
@@ -1029,6 +1029,8 @@
...
+      "DeepsightHarmonizable": "Deepsight Harmonizable",
```
Comment (bhollis) [inline config/i18n.json:1033]: This could be "Harmonizable" to save space

File: config/i18n.json [hunk 3]
```diff
@@ -1044,7 +1046,9 @@
       "Recoil": "Recoil",
       "Power": "Power",
       "Airborne": "Airborne"
-    }
+    },
+    "Yes": "Yes",
+    "No": "No"
```
Comment (bhollis) [inline config/i18n.json:1051]: For other boolean columns (e.g. sunset) we use a check icon or nothing, not yes and no.

File: src/app/organizer/Columns.tsx [hunk 4]
```diff
@@ -535,6 +536,21 @@ export function getColumns(
...
+        header: t('Organizer.Columns.DeepsightLevel'),
+        value: (item) => item.craftedInfo?.level,
+        defaultSort: SortDirection.DESC,
+      }),
+    destinyVersion === 2 &&
+      isWeapon &&
+      c({
+        id: 'deepsightHarmonizable',
+        header: t('Organizer.Columns.DeepsightHarmonizable'),
+        value: (item) => (isHarmonizable(item) ? t('Organizer.Yes') : t('Organizer.No')),
```
Comment (bhollis) [inline src/app/organizer/Columns.tsx:552]: The value should be the boolean (true or false) and then the cell should be the actual content.

**Review by bhollis — CHANGES_REQUESTED** [review]
Thanks! Left some comments about changes that are required before merging.

**Comment by bjacobel** [comment]
Thanks for the review! Those changes should be ready:

image

**Review by bhollis — APPROVED** [review]

**Pull request merged**